_inc/client/lib/analytics/index.js

```diff
@@ -55,12 +56,17 @@ function buildQuerystringNoPrefix( group, name ) {
 
 const analytics = {
 
-	initialize: function( userId, username, superProps ) {
+	initialize: function( blogid, userId, username, superProps ) {
```

Comment:
This is changing the function signature and moves all the parameters, do you think we could add the new parameter to the end, or at least move it right before `superProps`?

Reply:
addressed below